src/styles/hero.scss

```diff
@@ -0,0 +1,11 @@
+.hero-logo-lockup {
+  width: 400px;
+  fill: var(--grav-co-grp-b-neutral);
+}
+
+.hero-img {
+  background: url(../images/hero-test.jpg);
+  background-size: cover;
+  min-height: 450px;
+  width: 100%;
+}
```

Comment:
We could use an inline image for this like the Gravity homepage hero pattern? http://style-next.buildit.digital/components/detail/page-homepage.html

Reply:
oh yeah, not seen the `object-fit` css prop before - nice